crates/interpreter/src/exec.rs

```diff
@@ -1061,14 +1061,15 @@ impl<'m> Thread<'m> {
         let frame = self.frame();
         let label = frame.labels.pop().unwrap();
         if frame.labels.is_empty() {
-            let frame = self.frames.pop().unwrap();
-            debug_assert_eq!(frame.labels_values.len(), label.values_cnt);
-            debug_assert_eq!(frame.labels_values.len(), frame.arity);
+            let mut frame = self.frames.pop().unwrap();
+            frame.values.drain(0 .. frame.locals_cnt);
```

Comment:
I'm not sure what's the best here. We might want to try both options (and look what others are doing). It looks like we can either:
- keep the locals out of the stack (thus saving this drain)
- keep the locals inside the stack (thus saving popping them when calling a function)

The main difference is that draining is proportional in the number of remaining elements, while popping is proportional to what is popped. But I expect both of those numbers to be rather small, so maybe it doesn't matter. Just something to keep in mind once we have something working and start doing benchmarks.

Reply:
Thanks for the explanation. I'll add a TODO for this tradeoff in the next PR.